runtimes/datascience/ubi9-python-3.12/pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
     'scipy~=1.15.2',
     'skl2onnx~=1.18.0',
     'onnxconverter-common~=1.13.0', # Required for skl2onnx, as upgraded version is not compatible with protobuf
-    'codeflare-sdk~=0.29.0',
+    "codeflare-sdk~=0.29.0; platform_machine != 'ppc64le'",
```

Comment:
@coderabbitai we should determine whether to use ' or " for deps in pyproject.toml

create github issue and make a suggestion